crates/ruff_linter/resources/test/fixtures/refurb/FURB169.py

```diff
@@ -26,6 +26,19 @@
 
 type(None) != type(None)
 
+type(a.b) is type(None)
+
+type(
+    a(
+        # Comment
+    )
+) != type(None)
+
+type(
+    a := 1
+) == type(None)
+
```

Comment:
It doesn't seem those get reported in the snapshot. Would you mind taking a look?